samples/opencl/FindOpenCL.cmake

```diff
@@ -0,0 +1,174 @@
+# Find OpenCL
```

Comment:
Anyway I suggest to drop this file and to use official CMake module:
http://www.cmake.org/cmake/help/v3.1/module/FindOpenCL.html